types/ej.reports.all/tslint.json

```diff
@@ -0,0 +1,44 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
New definitions cannot have any of the rules disabled. Please fix all the errors and don't disable any rules.